.editorconfig

```diff
@@ -0,0 +1,18 @@
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
+end_of_line = lf
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+
+[{Makefile, *.sh}]
+indent_style = tab
+
+# Trailing spaces in markdown indicate word wrap
+[{*.markdown,*.md}]
+trim_trailing_spaces = false
+max_line_length = 80
```

Comment:
What is the effect of this `max_line_length`? Will longer lines be auto-wrapped?

Reply:
This is from the editorconfig's [default values](https://github.com/editorconfig/editorconfig-defaults/blob/master/.editorconfig#L13). I just copied it here, so we don't have any surprises with implicit configurations.

> Will longer lines be auto-wrapped?

VS code only supports a small subset of editorconfig parameters: https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig#supported-properties 
So the answer is: no, it is not supported by the vscode/plugin.